init_db.sh

```diff
@@ -10,11 +12,6 @@
 #   /> cf push my_app_name -c "null"
 # To find out more go here: https://docs.cloudfoundry.org/devguide/deploy-apps/start-restart-restage.html
 
-echo "------ Checking Super User Environment Variables are Set ------"
-"${OAUTH2_SUPER_USER:?You need to set the OAUTH2_SUPER_USER environment variable to something.}"
```

Comment:
This was failing although the variables were set so don't think it was checking it properly

Reply:
If I remember correctly this was what Mark wanted.
He specifically wanted the docker container to NOT start if the super user was not created for the OAuth2 server. Have a check with him first before you make this change. :-)

Kind regards, Nicholas.

Reply:
Thanks for the feedback @nherriot. The script isn't working for two reasons.
1. The check to see if the value is set is actually erroring because it's trying to execute the environmental variable. I'll look to move this into the python script
2. Even if there is an error the script will carry on because the script isn't set to exit if a command fails.

Reply:
Has the deploy process changed then? As the system needs those env variables to start. So you really need a way to log that error message. (i.e. you need to know 'why' it failed).
Yip good idea to do it all in python - good job to get this done! :-)

You probably know this but just in case:
https://pypi.org/project/shell/

Or you can use the plane old 'os' module:
https://docs.python.org/3/library/os.html

Keep up the good work. You guys are doing a good job. :-)